### DISCUSSION TITLE
How can I get player's msg toggle status?

### DISCUSSION THREAD

**Discussion by PJM03** [Zrips/CMI#4905] [Q&A]
like a title, how can I get player's msg toggle status?
as an example, player's teleport toggle status use `CMIUser#isTeleportToggled` to get status. but `CMIUser` don't have method like `isMsgToggled`. how can I get player's msg toggle status?

(I don't good at English. if you can't understand my question, please let me know.)

**Comment by Zrips** [comment]
CMIUser#isAcceptingPM
Maybe some what confusing name to have, but its there.

**Reply by PJM03** [reply]
Ah, thank you for your answer! :)
